_data/authors.yml

```diff
@@ -260,3 +260,10 @@ Senpai1199:
   web:
   bio: |
     Tech enthusiast and a gamer!
+
+barhouum7:
```

Comment:
Your name should already be in this file. Make sure to sync with master just to be sure.

Reply:
I know I added it before, but it is not there anymore, and I don't know why!!